.golangci.yml

```diff
@@ -1,21 +1,18 @@
 linters:
   enable:
-    - deadcode
     - errcheck
     - goconst
     - gofmt
-    - golint
     - gosec
     - govet
     - ineffassign
-    - interfacer
     - lll
     - megacheck
     - misspell
     - nakedret
-    - structcheck
+    - revive
```

Comment:
The changes here allow the linter to trigger without warnings on deprecated linters.